.github/workflows/docker-build.yaml

```diff
@@ -27,6 +32,13 @@ jobs:
         with:
           context: .
           push: true
-          tags: "registry.scality.com/backbeat-dev/backbeat:${{ github.sha }}"
+          tags: ${{ env.REGISTRY_NAME }}/${{ env.PROJECT_NAME }}-dev/${{ env.PROJECT_NAME }}:${{ github.sha }}
           cache-from: type=gha
           cache-to: type=gha,mode=max
+
+      - name: Push dashboards to the development namespace
+        run: |
+          oras push ${{ env.REGISTRY_NAME }}/${{ env.PROJECT_NAME }}-dev/${{ env.PROJECT_NAME }}-dashboards:${{ github.sha }} \
```

Comment:
do we need to install oras somewhere or is it already present?

Reply:
Oras is part of the installed softwares: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md